sklearn/neural_network/tests/test_mlp.py

```diff
@@ -59,7 +60,7 @@ def test_alpha():
     absolute_sum = lambda x: np.sum(np.abs(x))
 
     for alpha in alpha_values:
-        mlp = MLPClassifier(hidden_layer_sizes=10, alpha=alpha, random_state=1)
+        mlp = MLPClassifier(hidden_layer_sizes=10, alpha=alpha, random_state=1, max_iter=500)
```

Comment:
@jnothman Because it was possible to suppress the warning by adapting the parameters. But then, it's true that I was not consistent with this thought everywhere possible. Do you think it is better to silence the warning explicitely in all cases or to adapt the classifier parameters such that the warning is not raised in the first place?